src/keys/bip39.rs

```diff
@@ -170,4 +187,21 @@ mod test {
             Mnemonic::generate((MnemonicType::Words24, Language::English)).unwrap();
         assert_eq!(generated_mnemonic.valid_networks, any_network());
     }
+
+    #[test]
+    fn test_mnemonic_to_master() {
```

Comment:
This is two tests, right? One tests mnemonic to master _with_ a passphrase and one tests it _without_ a passphrase. Perhaps it would be cleaner split into two?

Reply:
Yes should be two tests, fixed also.